website/static/js/osfHelpers.js

```diff
@@ -510,6 +510,20 @@ ko.bindingHandlers.listing = {
     }
 };
 
+/* Responsive Affix for side nav */
+function fixAffixWidth() {
+    $('.affix, .affix-top, .affix-bottom').each(function (){
+        var el = $(this);
+        var colsize = el.parent('.affix-parent').width();
+        el.outerWidth(colsize);
+    });
+}
+
+var initializeResponsiveAffix = function (){
+    $(window).resize(function (){ fixAffixWidth(); });
```

Comment:
@caneruguz @GageGaskins I believe the `resize` event gets called continuously as the window is resized. This can get to be pretty expensive. You will likely need to throttle or debounce the calls to `fixAffixWidth`.